pkgs/by-name/wo/wooting-udev-rules/package.nix

```diff
@@ -2,9 +2,9 @@
 
 stdenv.mkDerivation rec {
   pname = "wooting-udev-rules";
-  version = "unstable-2023-03-31";
+  version = "unstable-2024-11-20";
```

Comment:
Nit: Package versions are now supposed to be prefixed with `0-unstable...` so when an eventual release (1.x) comes it'll be picked up as newer.

Reply:
Ah good to know

Reply:
it should be fine here, as there is no dedicated package containing these udev rules specifically and not looking like there ever will be one. And its previously been like this too. So not a blocker in my opinion, but worth mentioning.

Reply:
Fixed it